src/main/java/net/datafaker/transformations/sql/SqlTransformer.java

```diff
@@ -219,19 +219,16 @@ private String handleObject(Object value) {
     }
 
     private String handledObjectToString(int length, String strValue) {
-        StringJoiner joiner = null;
+        StringBuilder builder = new StringBuilder();
```

Comment:
May as well set the builder size to length to start with, that will probably mean less resizing

Reply:
yes ,It seems more reasonable to set the length

Reply:
is done

Reply:
I've made the change, thank you!